### PR TITLE
Addding my card.

### PR DESCRIPTION
**Pull request by hauphanlvc**
Signed-off-by: Hau Trong Phan <hauphanlvc@gmail.com>

### CODE REVIEW

**Comment by hauphanlvc** [comment]
Thanks for helping us to have a place to get the first contribution.

File: index.html [hunk 1]
```diff
@@ -121,6 +121,25 @@ <h2>A project for first-time contributions</h2>
...
+            Hi, I'm just a person who want to know how to contribute a project, and become a professional full stack developer. 
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://free-for.dev/#/" target="_blank" title="First Resource">Get all free services </a>
+              </li>
+            </ul>
+          </div>
```
Comment (theAravinthM) [inline index.html:141]: In this resources section, providing 3 helpful online resources is a good practice! It seems you have provided 1 resource link, which is fine. But, adding two more is recommended. 

You could add any resource link that the fellow developers could make use of !

**Review by theAravinthM — CHANGES_REQUESTED** [review]
Hi @hauphanlvc Happy that you made a Pull Request and made your first open source contribution! 
Your PR needs a slight change to get merged! Get back with the updations sooner to successfully make your PR merged!

**Comment by hauphanlvc** [comment]
I have added more new resources, please help me to check it. Thank you !!!

**Comment by theAravinthM** [comment]
Congratulations on your PR Merge, and Thanks for your Contribution! Your card will be live on our Website in a moment! 🚀

Now you can follow the Next Steps. You are also welcome to join our Discord Server: https://discord.gg/tWkvS4ueVF and be a part of our community.

Happy contributing!

**Pull request merged**